google-cloud-bigtable/google-cloud-bigtable.gemspec

```diff
@@ -20,7 +20,8 @@ Gem::Specification.new do |gem|
 
   gem.required_ruby_version = ">= 2.0.0"
 
-  gem.add_dependency "google-gax", "~> 0.10.1"
+  gem.add_dependency "google-gax", "~> 1.0.0"
```

Comment:
Should the dependencies be changed to `~> 1.0`? Is your intent to disallow use of google-gax version 1.1.0?

Reply:
Whoops, updated.

Reply:
What about google-cloud-core, should it remain `~> 1.0.0`?

Reply:
Yes, this should be changed for all the google-gax dependencies. Once a library that follows SEMVER is moved to 1.0 we should always allow new minor versions as it should be backwards compatible on the major version.

Reply:
Done.